src/components/modals/ModalHeader.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './ModalHeader.css';
+
+const ModalHeader = ({ children, onRequestClose, ...rest }) => {
+  return (
+    <div className="ReactModal__Header" {...rest}>
+      {children}
+      {onRequestClose && (
```

Comment:
I think this is weird that we rely on showing something by passing handler function. I think it would be better to be more verbose here and add prop `shouldDisplayCloseButton` or something like that